src/main/java/io/github/externschool/planner/entity/profile/Person.java

```diff
@@ -0,0 +1,119 @@
+package io.github.externschool.planner.entity.profile;
+
+import io.github.externschool.planner.entity.User;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "person")
+@Inheritance(strategy = InheritanceType.JOINED)
+public class Person {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
```

Comment:
@Column(name = "id", updatable = false, nullable = false)
if you like these recommendations: https://www.thoughts-on-java.org/jpa-generate-primary-keys/